testing/scenarios/custom_todos/standart/main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
The directory should be `standard` not `standart`

Reply:
Renamed the directory.